### ISSUE TITLE
Cannot run vue demo

### ISSUE THREAD

**Issue by tomwayson** [Esri/arcgis-rest-js#268]
It looks like we need to either revlock the version of webpack, or install webpack-cli. Following the instructions and running `npm start` I get:

```bash
➜  vue git:(master) ✗ npm start

> @esri/arcgis-rest-demo-vue-with-popup@1.6.0 start /Users/tom6947/code/arcgis-rest-js/demos/vue
> npm run dev


> @esri/arcgis-rest-demo-vue-with-popup@1.6.0 dev /Users/tom/code/arcgis-rest-js/demos/vue
> cross-env NODE_ENV=development webpack-dev-server --open --hot

The CLI moved into a separate package: webpack-cli.
Please install 'webpack-cli' in addition to webpack itself to use the CLI.
-> When using npm: npm install webpack-cli -D
-> When using yarn: yarn add webpack-cli -D
module.js:540
    throw err;
    ^

Error: Cannot find module 'webpack-cli/bin/config-yargs'
```

OK, I'll bite, `npm i webpack-cli`, then `npm start`:

```bash
➜  vue git:(master) ✗ npm start

> @esri/arcgis-rest-demo-vue-with-popup@1.6.0 start /Users/tom/code/arcgis-rest-js/demos/vue
> npm run dev


> @esri/arcgis-rest-demo-vue-with-popup@1.6.0 dev /Users/tom/code/arcgis-rest-js/demos/vue
> cross-env NODE_ENV=development webpack-dev-server --open --hot

/Users/tom/code/arcgis-rest-js/demos/vue/node_modules/webpack-dev-server/bin/webpack-dev-server.js:405
    throw e;
    ^

TypeError: Cannot read property 'compile' of undefined
```

You're dead to me webpack!

**Comment by jgravois** [comment]
@itrew can you spare any cycles to help us make sure this demo app stays plug and play?

**Comment by itrew** [comment]
@jgravois Sure, I can take a look at this then.

**Comment by jgravois** [comment]
resolved by #279. thank you @itrew!
